srcpkgs/spotify-adblock/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'spotify-adblock'
+pkgname=spotify-adblock
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+build_helper=rust
+conf_files="/etc/spotify-adblock/config.toml"
+hostmakedepends="make cargo"
+makedepends="rust-std"
+short_desc="Adblocker for Spotify"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/abba23/spotify-adblock"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
expand `$homepage` here

Reply:
Why? Why is it needed to expand `$homepage`?

Reply:
also use `archive/refs/tags/`

Reply:
if $homepage changes in the future, it means distfiles needs to change too. it's kinda seen as frowned upon by some maintainers: #27735